pom.xml

```diff
@@ -902,6 +902,71 @@
 
 			</properties>
 		</profile>
+		        
+		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
+        <profile>
+			<id>sigmah4hi</id>
```

Comment:
Just to finalize this topic, would "sigmah-userbuild-*" also do the job ?
I would prefer that because organisation which will request modification of
the compilation just for their needs won't often be UN agencies or members
of Red Cross/Red Crescent movement. And they aren't NGOs.
Is that ok ?

To move forward, to I need to cancel this pull request, modify my codebase
and make another pull request ?


*--*Blog : http://www.grattecielhorizontal.info/
*  dernier post: *Idée d’un autre rapport de force face à la COP21
<http://www.grattecielhorizontal.info/Idee-d-un-autre-rapport-de-force-face-a-la-COP21>

2016-01-29 22:24 GMT+01:00 Raphaël <notifications@github.com>:

> In pom.xml
> <https://github.com/sigmah-dev/sigmah/pull/4#discussion_r51317676>:
>
> > @@ -902,6 +902,71 @@
> >
> >  			</properties>
> >  		</profile>
> > +		
> > +		<!-- ========= [ HANDICAP INTERNATIONAL SPECIAL VERSION. ] ========= -->
> > +        <profile>
> > +			<id>sigmah4hi</id>
>
> I agree with numero-six, the prefix sigmah-ngo-* is more user friendly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sigmah-dev/sigmah/pull/4/files#r51317676>.
>